src/API/Model/ResultsMetadata.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace GroupByInc\API\Model;
+
+class ResultsMetadata
+{
+  /**
+   * @var boolean
+   */
+  private $recordLimitReached;
+
+  /**
+   * @var long
+   */
+  private $totalTime;
+
+  /**
+   * @return boolean  True if the total record count is actually greater than the returned record count in the response, false otherwise
+   */
+  public function hasRecordLimitReached()
```

Comment:
@grrilla Suggestion; this method name may be more clearly named as `hasReachedRecordLimit()`

Reply:
```recordLimitReached``` is the name of a member we use on this object across APIs. I named it this way to abide convention - it just comes out semantically weird in PHP which prefixes with ```is``` or ```has``` instead of ```get```. I'm open to changing this but I don't like to break convention or consistency